### PR TITLE
adding the who is missing feature and changing the message when we co…

### PR DESCRIPTION
**Pull request by monoguerin**
…mplete hours

### CODE REVIEW

File: src/util/sendMessage.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                '?date=' + lastBusinessDay.format('YYYY-MM-DD');
+            } else {
+                message = '¡Hemos Cumplido! 🙌 \n\n' +
+                'https://1.bp.blogspot.com/-cQ0b6am0ckc/WIEbGsVjH2I/AAAAAAAAAMI/' +
+                '8W0wdpRdGNYCOfU2I1TD3NK2mIbqiI3QACLcB/s320/hemoscumplido.jpg';
+            }
+
+            const msg = new builder.Message()
+            .address(subscription)
+            .text('¡Buenos Días!\n\n' + message );
```
Comment (andreszorro) [inline src/util/sendMessage.js]: I'd make this message parametrized, so it just says "Buenos días" when the message is programmed, and not when requested.

Comment (monoguerin) [inline src/util/sendMessage.js]: will do...

Comment (andreszorro) [inline src/util/sendMessage.js]: We could even fetch the requester name, like "Hola, César. Las personas que faltan por reportar son:"

File: src/util/sendMessage.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+            if (!data) {
+                message = 'Hubo un error al llamar al API de Penguin Report. 😅';
+            }
+
+            const penguined = data.filter((peep) => peep.totalHours < 7)
+            .map((peep) => `🐧  <b>${peep['person-name']}</b> (${peep.totalHours} horas)`);
+
+            if (penguined.length) {
+                message = 'Las siguientes personas aún no han reportado horas:\n\n' +
```
Comment (oagarcia) [inline src/util/sendMessage.js]: @andreszorro @monoguerin Not sure if the project enforces using template literals. If is the case, worth to refactor these lines.

Comment (andreszorro) [inline src/util/sendMessage.js]: I left it loose, to avoid things like odd file indenting on messages that contained returns.

**Pull request merged**